packages/adders/lucia/index.ts

```diff
@@ -217,6 +217,9 @@ export default defineAdder({
 				});
 				imports.addNamed(ast, '@oslojs/crypto/sha2', { sha256: 'sha256' });
 				imports.addNamed(ast, 'drizzle-orm', { eq: 'eq' });
+				if (typescript) {
```

Comment:
I'm not sure if this is the correct way to add type imports

Reply:
this is good 👍